v2rayN/v2rayN/Handler/ConfigHandler.cs

```diff
@@ -1178,6 +1178,58 @@ static bool Contains(string str, params string[] s)
             }
 
             ProfileItem profileItem = new();
+
+            //Is v2ray array configuration
+            var v2rayArrayConfig = JsonUtile.Deserialize<V2rayConfig[]>(clipboardData);
```

Comment:
Please do not use `V2rayConfig `to parse, use `object `instead
This ensures that all the user's custom configurations are obtained

Reply:
You are right. It is better to store what obtained from user.